circular_linked_list/circular_linked_list.cpp

```diff
@@ -0,0 +1,44 @@
+#include<iostream>
+using namespace std;
+struct node{
+    int data;
+    node* next;
+};
+
+void show(node *head) {
+    node *current = head;
+    cout<<"The list is ";
+    do {
+        cout << current->data << " ";
+        current = current->next;
+    } while(current != head);
```

Comment:
Missing space before ( in while(  [whitespace/parens] [5]

*Origin: CPPLintBear, Section: `all.cpp`.*